### ISSUE TITLE
Add GraphQL API

### ISSUE THREAD

**Issue by saerdnaer** [crs-tools/tracker#196]
As follow up to #185 and in context of https://media.ccc.de/v/geekend2018-5-graphql – what are your options on adding a GraphQL API to the tracker? 

It looks like there exist the corresponding libraries for PHP (e.g. 
https://github.com/webonyx/graphql-php ) and here is an example GraphQL server implementation https://github.com/benawad/php-graphql-example ( explained in https://www.youtube.com/watch?v=jRuSicPIeUY )

**Comment by jjeising** [comment]
Yes, I've already thought about GraphQL. It requires more implementation work on the server, but seems very flexible.
